flex/http.py

```diff
@@ -218,9 +241,27 @@ def _normalize_urllib_response(response, request=None):
     )
 
 
+def _normalize_tornado_response(response, request=None):
+    if not _tornado_available:
+        raise TypeError("Tornado is not installed")
+
+    if not isinstance(request, tornado.httpclient.HTTPResponse):
+        raise TypeError("Cannot normalize this request object")
+
+    return Response(
+        request=request,
+        content=response.body,
+        url=response.effective_url,
+        status_code=response.code,
+        content_type=response.headers.get('Content-Type'),
+        response=response,
+    )
+
+
 RESPONSE_NORMALIZERS = (
     _normalize_urllib_response,
     _normalize_requests_response,
+    _normalize_tornado_response,
```

Comment:
(arbitrary thread anchor)

I managed to get tests running (after chasing down the extra requirements; we we should add those to requirements.txt), but am getting a few errors from `fixture 'httpbin' not found`. How do I fix that?